README.md

```diff
@@ -1,70 +1,48 @@
-# Getting Started with Create React App
+# Math magician
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> Handy Calculator App
 
-## Available Scripts
+![image](https://user-images.githubusercontent.com/82236725/162061689-59a0aa7e-4be1-42dd-a9f1-70784bfd4195.png)
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- Html and Css
+- Javascript
+- Github and GitFlow
+- React
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## How to run in your local machine
 
-### `npm test`
+- Copy the URL: git@github.com:Sirvelasque/Math_Magician.git"
+- In your terminal, go to the directory you want to clone the repository.
+- Use the command: git clone git@github.com:Sirvelasque/Math_Magician.git
+- Open index.html file
+- Run npm install in the terminal to install node modules
+- Execute npm run dev in terminal to build the development files
+- To start a server run npm start and the server would be started
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `npm run build`
+## Authors
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+👤 **Walter Velasquez**
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- GitHub: [@Sirvelasque](https://github.com/Sirvelasque)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
-### `npm run eject`
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## 🤝 Contributing
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+Contributions, issues, and feature requests are welcome!
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] [optional] It would be nice to add a link to your issue page so that people would make contributions to your project.